### PR TITLE
changes to the benchmark.ipynb file to work on GPUs

### PR DESCRIPTION
**Pull request by dominikhoeing**
changes to the benchmark notebook include:

1. I moved up the cell containing the creation of `eval_dataset` and `eval_dataloader`
2. I added a cell that creates a variable called `device`, which will select the GPU if available and else the CPU and it moves the model to the device
3. I have edited the cell containing the training of the model. It now creates an additional dictionary called `val_loss` for tracking the loss of the test set (look for part containing `eval_dataloader`)
4. After training the model I added a cell that moves the model and data back to the CPU and empties the cache of the GPU.
5. I have added a cell that plots the validation loss. Beware: I was lazy and didn't change the variable name used for plotting (`train_loss`)
6. I also added a cell printing the validation loss. (same remark as above about the variable name)

### CODE REVIEW

**Review by TetyanaSam — APPROVED** [review]
thank you for gpu and validation los
